.github/dependabot.yml

```diff
@@ -34,11 +34,13 @@ updates:
   directory: "/modules/engage-paella-player"
   schedule:
     interval: daily
+  target-branch: "r/11.x"
```

Comment:
A note to update this should be added to the release manager notes

Reply:
Solved in new commit